Source/WebCore/css/typedom/StylePropertyMap.cpp

```diff
@@ -58,6 +58,18 @@ static RefPtr<CSSValue> cssValueFromStyleValues(std::optional<CSSPropertyID> pro
     return list;
 }
 
+static bool propertyAllowsNumberOrIntegerInput(CSSPropertyID propertyID)
+{
+    if (CSSProperty::allowsNumberOrIntegerInput(propertyID))
+        return true;
+    // Properties using custom parsing code (which don't have a grammar rule in CSSProperties.json.
+    return propertyID == CSSPropertyBorderImageOutset
```

Comment:
I think a better approach for this would be to add a new `codegen-properties` property like "custom-parser-allows-number-or-integer-input": true, for these remaining properties. That way people don't accidentally call the generated one and get confused.

Reply:
OK, I can certainly do that. I'll update the patch today.